### DISCUSSION TITLE
ENOENT: no such file or directly

### DISCUSSION THREAD

**Discussion by sasikumarevature** [community/community#136456] [Programming Help]
### Select Topic Area

Show & Tell

### GitHub Feature Area

VSCode

### Body

ENOENT: no such file or directly, open ‘/workspace/projetc/.git/refs/remotes/origin/main
Why I am getting this this error when I try to commit it  ?

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @sasikumarevature ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
